components/company/CompanyLogo.tsx

```diff
@@ -1,13 +1,23 @@
+import { Avatar } from 'antd';
 import React from 'react';
```

Comment:
Nit: We've been importing `useState` instead of the whole React namespace throughout the codebase, shall we stick to that?
```suggestion
import { useState } from 'react';
```